content/en/docs/tasks/administer-cluster/dns-horizontal-autoscaling.md

```diff
@@ -148,7 +132,7 @@ Modify the fields according to your needs. The "min" field indicates the
 minimal number of DNS backends. The actual number of backends number is
 calculated using this equation:
 
-    replicas = max( ceil( cores * 1/coresPerReplica ) , ceil( nodes * 1/nodesPerReplica ) )
+    replicas = max( ceil( cores × 1/coresPerReplica ) , ceil( nodes × 1/nodesPerReplica ) )
```

Comment:
@sftim . Changes seem fine. Small nit as to whether line 135 change is needed.
/lgtm

Reply:
I'd use MathML if I had a chance!

Reply:
/approve